From 6e6643d1def3ba50c4da492d05ddc53c599df3dc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Erick=20P=C3=A9rez=20Castellanos?= <erick.red@gmail.com>
Date: Fri, 13 Feb 2015 12:35:00 -0500
Subject: build: bump glib requirements

This is needed for the use of G_DECARE_FINAL_TYPE macros

diff --git a/configure.ac b/configure.ac
index 05dd33d..1f6f8d5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -81,7 +81,7 @@ MAINTAINER_COMPILER_FLAGS="\
 MAINTAINER_COMPILER_FLAGS=${MAINTAINER_COMPILER_FLAGS#*  }
 AC_SUBST(MAINTAINER_COMPILER_FLAGS)
 
-GLIB_MIN_VERSION=2.43.2
+GLIB_MIN_VERSION=2.43.4
 GTK_MIN_VERSION=3.15.4
 ECAL_REQUIRED=3.13.90
 EDATASERVER_REQUIRED=3.13.90
-- 
cgit v0.10.2

